### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by laurentpellegrino** [oembedler/graphql-spring-boot#7]
@oembedler The last commit is 7 months old and recent pull requests have no comments. Is the project still maintained?

**Comment by asterisk622** [comment]
I have the same question

**Comment by asterisk622** [comment]
@oembedler  Is the project still maintained? This is very useful project

**Comment by shishuwu** [comment]
Agree, very good project, hope it is still maintained... @oembedler

**Comment by ryangardner** [comment]
It seems pretty safe to say that this project is _not_ actively maintained. (it's been months since someone asked the question if it is maintained or not... and no response)

Has anyone considered forking this to a new org and taking on a stewardship / maintenance role with it?

**Comment by apottere** [comment]
Because of inactivity, we've forked it into graphql-java and we'll maintain it there till (if ever) the maintainer returns for it: https://github.com/graphql-java/graphql-spring-boot

**Comment by shishuwu** [comment]
Bravo!!! @apottere 
Really appreciate it.
